rollup.config.js

```diff
@@ -45,13 +45,13 @@ const plugins = (minify) =>
 
 export default [
   // UMD build
-  // dist/chart.umd.js
+  // dist/chart.umd.min.js
   {
     input: 'src/index.umd.ts',
     plugins: plugins(true),
     output: {
       name: 'Chart',
-      file: 'dist/chart.umd.js',
```

Comment:
this will be breaking for people who have this referenced in their code, I think you need to add a second build to the output like I added here before I noticed this PR

https://github.com/chartjs/Chart.js/pull/11458

Reply:
Thanks for the note, updated at 204abcc46c341ad086eb75f06a9604d80dd40ecb